guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -37,6 +37,19 @@ ifdef::katello,satellite[]
 == Configuring {SmartProxyServer} with SSL Certificates
 endif::[]
```

Comment:
This endif is looking weird. Maybe it would be better to leave out both conditions below and add the text of the second one under the condition above?